src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -32,8 +32,18 @@ class ObjectField extends React.Component {
     super();
     this.isRequired = this.isRequired.bind(this);
     this.orderAndFilterProperties = _.flow(
-      properties =>
-        orderProperties(properties, _.get('ui:order', this.props.uiSchema)),
+      properties => {
+        const order = _.get('ui:order', this.props.uiSchema);
+        const filtered = order
+          ? // If there is no ui:order, don't bother filtering it; leave it as
+            // undefined so orderProperties doesn't get angry about missing
+            // properties
+            order.filter(prop =>
+              Object.keys(this.props.schema.properties).includes(prop),
+            )
+          : order;
```

Comment:
Could we not bother checking order and simply use optional chaining, i.e. `order?.filter(...)` ?

Reply:
Ahhhh, duh. You're absolutely right. Optional chaining would do it nicely here.

Look, I haven't touched code for over a month, man! :facepalm: :sweat_smile: